demo/src/index.ts

```diff
@@ -1,4 +1,8 @@
+import './test.scss';
+import './base.css';
+
 declare const PACKAGE_VERSION: string;
 
 console.log('I am a test build file');
 console.log(PACKAGE_VERSION);
+
```

Comment:
Where can I view this?

Reply:
`npm run demo` or `npm run demo:build`

Reply:
Is see the provider and client windows, but not this one?

Reply:
Sorry had multiple branches testing different things with the demo folder. Should be fixed now.